### PR TITLE
issue #3834 removed references to Db2 in documentation

### PR DESCRIPTION
**Pull request by punktilious**
Signed-off-by: Robin Arnold <robin.arnold@ibm.com>

### CODE REVIEW

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -1742,15 +1708,15 @@ This feature is useful for imports which follow a prefix pattern:
 
 
 ### 4.10.3 Integration Testing
-To integration test, there are tests in `ExportOperationTest.java` in `fhir-server-test` module with server integration test cases for system, patient and group export. Further, there are tests in `ImportOperationTest.java` in `fhir-server-test` module. These tests rely on the `fhir-server-config-db2.json` which specifies two storageProviders.
+To integration test, there are tests in `ExportOperationTest.java` in `fhir-server-test` module with server integration test cases for system, patient and group export. Further, there are tests in `ImportOperationTest.java` in `fhir-server-test` module. These tests rely on the `fhir-server-config-postgres.json` which specifies two storageProviders.
```
Comment (punktilious) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:1711]: Need to update this because it no longer reflects the fhir-server-test behavior.

Comment (lmsurpre) [reply docs/src/pages/guides/FHIRServerUsersGuide.md:1711]:
```suggestion
There are tests in the `fhir-server-test` module for system, patient and group export. These tests rely on fhir-server-config files that currently exist under `fhir-server-webapp/src/test/liberty`.
```

Comment (lmsurpre) [reply docs/src/pages/guides/FHIRServerUsersGuide.md]: new text looks good

File: fhir-install/src/main/docker/ibm-fhir-schematool/DESIGN.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The design is such that it COULD, but does not support multiple input files at t
 When the container is started, the container process one of two flows:
 
 * Schema Onboarding - creates and updates a schema and apply grants
-* Schema Offboarding - deletes a tenant (Db2) and removes data if there are no tenants left.
+* Schema Offboarding - drops a schema
```
Comment (lmsurpre) [inline fhir-install/src/main/docker/ibm-fhir-schematool/DESIGN.md:17]: not a comment on this line, but a comment about this ibm-fhir-schematool design in general:   I wonder if we should remove all fancy capabilities from this image and have it be just a simple container wrapper around the cli like our other images?

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**